src/calculator/votes.ts

```diff
@@ -17,8 +18,8 @@ export async function getVotesWithCoefficients(
   options: {
     minimumAmountUSD?: number;
     enablePassport?: boolean;
-    passportThreshold?: number;
-  }
+  },
+  pmOptions: ProportionalMatchOptions
```

Comment:
```suggestion
  proportionalMatchOptions: ProportionalMatchOptions
```

I would suggest to have the full name here